presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/TestElasticsearchIntegrationSmokeTest.java

```diff
@@ -684,6 +684,31 @@ public void testMixedCase()
                 "VALUES ('john', 32)");
     }
 
+    @Test
+    public void testNumericKeyword()
+            throws IOException
+    {
+        String indexName = "numeric_keyword";
+        @Language("JSON") String mapping = "" +
+                "{" +
+                "  \"mappings\": {" +
+                "    \"doc\": {" +
+                "      \"properties\": {" +
+                "        \"numeric_keyword\":   { \"type\": \"keyword\" }" +
+                "      }" +
+                "    }" +
+                "  }" +
+                "}";
+        createIndex(indexName, mapping);
+        index(indexName, ImmutableMap.<String, Object>builder()
+                .put("numeric_keyword", 20)
+                .build());
+
+        assertQuery(
+                "SELECT numeric_keyword FROM numeric_keyword",
+                "VALUES (20)");
```

Comment:
This can be written as `VALUES 20`. The parenthesis are unnecessary.

Reply:
Let's add another query that filters based on such a field to ensure ES can match the values properly:

```
SELECT numeric_keyword FROM numeric_keyword WHERE numeric_keyword = '20'
```